metaflow/tutorials/00-helloworld/hello.py

```diff
@@ -21,6 +21,7 @@ def start(self):
         self.x = 100
         self.next(self.hello)
 
+    @resources(cpu=0.5, memory=150)
```

Comment:
For manual test only. This change is not necessary. Do we have an easy way to unit test compiled KFP yaml output?